pyop2/types/mat.py

```diff
@@ -618,11 +620,16 @@ def _init_monolithic(self):
         rset, cset = self.sparsity.dsets
         rlgmap = rset.unblocked_lgmap
         clgmap = cset.unblocked_lgmap
-        mat.createAIJ(size=((self.nrows, None), (self.ncols, None)),
-                      nnz=(self.sparsity.nnz, self.sparsity.onnz),
-                      bsize=1,
-                      comm=self.comm)
+        if self.mat_type == "is":
+            # FIXME monolithic lgmaps
```

Comment:
@pbrubeck Is this still broken? We can do mixed Poisson  H(div) x L2 if we have monolithic l2gmaps

Reply:
This should fix the tests https://gitlab.com/petsc/petsc/-/merge_requests/8705
Is this comment  `# FIXME monolithic lgmaps` outdated?

Reply:
The LGMaps seem correct as the test without bcs is passing